GraphConsoleAppV3/GraphRequests.cs

```diff
@@ -0,0 +1,1534 @@
+                newUser.UserType = "Member";
+
+                try
+                {
+                    await newUser.UpdateAsync();
+                    Console.WriteLine("\nUser {0} was updated", newUser.DisplayName);
+                }
+                catch (Exception e)
+                {
+                    Program.WriteError("\nError Updating the user {0} {1}", e.Message,
+                        e.InnerException != null ? e.InnerException.Message : "");
+                }
+            }
+
+            #endregion
+        }
+
+        private static async Task ResetUserPassword(IUser newUser)
+        {
+            #region Reset password for user
+
+            //*******************************************************************************************
+            // update the newly created user's Password and PasswordPolicies
+            // requires Directory.AccessAsUser.All and that the current user is a user, helpdesk or company admin
+            //*********************************************************************************************
+            if (newUser.ObjectId != null)
+            {
+                // update User's password policy and reset password - forcing change password at next logon
+                PasswordProfile PasswordProfile = new PasswordProfile
+                {
+                    Password = "changeMe!",
+                    ForceChangePasswordNextLogin = true
+                };
+                newUser.PasswordProfile = PasswordProfile;
+                newUser.PasswordPolicies = "DisablePasswordExpiration, DisableStrongPassword";
+                try
+                {
+                    await newUser.UpdateAsync();
+                    Console.WriteLine("\nUser {0} password and policy was reset", newUser.DisplayName);
+                }
+                catch (Exception e)
+                {
+                    Program.WriteError("\nError Updating the user {0} {1}", e.Message,
+                        e.InnerException != null ? e.InnerException.Message : "");
+                }
+            }
+
+            #endregion
+        }
+
+        private static async Task OtherUserWriteOperations(IActiveDirectoryClient client, IUser newUser,
+            IUser signedInUser)
+        {
+            #region Other user write operations on a newly created user
+
+            // *************************************************************
+            // These operations require more privileged permissions like Directory.ReadWrite.All or Directory.AccessAsUser.All
+            // Update signed in user's manager, update group membership
+            // **************************************************************
+
+            #region Assign a manager
+
+            // Assign the newly created user a new manager (the signed in user).
+            if (newUser.ObjectId != null)
+            {
+                Console.WriteLine("\n Assign User {0}, {1} as Manager.", signedInUser.DisplayName,
```

Comment:
reworded.

---
In reply to: [80543814](https://github.com/Azure-Samples/active-directory-dotnet-graphapi-console/pull/37#discussion_r80543814) [](ancestors = 80543814)